scout/commands/update/individual.py

```diff
@@ -13,6 +13,8 @@
     "tiddit_coverage_wig",
     "upd_regions_bed",
     "upd_sites_bed",
+    "splice_junctions_bed",  # An indexed junctions .bed.gz file obtained from STAR v2 aligner *.SJ.out.tab file.
```

Comment:
These comments (and the new keys) would look really neat in the individual model on models/case/case.py!

Reply:
Agreed!